cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/resource/PodOperations.java

```diff
@@ -37,7 +37,7 @@ public PodOperations(Vertx vertx, KubernetesClient client) {
      * Returns whether the pod given by {@code namespace} and {@code name} is ready
      * @param namespace The namespace.
      * @param name The name.
-     * @return True iff the pod is ready.
+     * @return True if the pod is ready.
```

Comment:
The iff was intentional, actually. It's short for "if and only if". But I accept that the casual ready wouldn't know that, and just plain if will suffice.

Reply:
I didn't know about that :-)

Reply:
Yeah, it's a maths thing which I never got out of the habit of using.